qiskit_sphinx_theme/pytorch_base/static/css/theme.css

```diff
@@ -10967,6 +10967,7 @@ article.pytorch-article .class .attribute dt {
   border-top: none;
 }
 article.pytorch-article .class .attribute dt em.property {
+  text-transform: capitalize;
```

Comment:
What happens if you set this to `none`? `capitalize` is not correct because it will capitalize things like `float` which should be lowercase.

Reply:
ah good point!

Reply:
`unset` would probably work  too, I'm not sure which is preferable in css 🤔

Reply:
See ChatGPT 4 below (lol). tl;dr is to use `none`.

--

> In CSS, I want to turn off a parent rule that is setting `text-transform: uppercase`. Is it better to use `unset` or to set `text-transform: none`? Please explain when it is appropriate to use one approach vs. the other.

In your situation, both unset and text-transform: none could potentially be used to override the parent rule, but their behavior can be different in certain scenarios. Let's delve into the specifics of both to understand when it's appropriate to use one over the other.

* text-transform: none - This property value will turn off the text transformation and display the text as it is. It is direct and explicit about not applying any transformation to the text.
* unset - The unset CSS keyword is used to reset a property to its inherited value if it inherits from its parent, and to its initial value if not. In the case of text-transform, the initial value is none, and the property is inheritable.

Now, in your case:

* If you use text-transform: none, it will always cancel out any transformation, whether it's from a parent element or not. It's very explicit.
* If you use unset, it will reset the property to its inherited value if it inherits from its parent, and to its initial value if not. If the parent element has a text-transform: uppercase rule, and the grandparent doesn't have any text-transform rule, unset will result in text-transform: none, since that's the initial value. However, if the grandparent had text-transform: lowercase, then unset would result in text-transform: lowercase, because that's the inherited value.

So, when to use one over the other?

* If you want to ensure the text is not transformed regardless of what any ancestor elements specify, use text-transform: none.
* If you want the element to respect the text-transform rule of its ancestors (if any), use unset.

Remember, unset can potentially inherit styles from further up the tree, not just the immediate parent. So if the intent is to specifically override the immediate parent's style and not necessarily to set it to the default, none would be the better choice.